project-reports/all.cfg

```diff
@@ -1,4 +1,3 @@
-[hyperledger]
```

Comment:
Think ConfigParser requires to have a header here ?

```python
MissingSectionHeaderError: File contains no section headers.
file: 'all.cfg', line: 1
'title = Hyperledger\n'
```

Reply:
done. Not sure what happened.